src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -252,4 +257,11 @@ public void proxyReload(@NotNull ProxyReloadEvent event) {
         reloadUpdate();
         plugin.log("Velocitab has been reloaded!");
     }
+
+    public void removeOfflinePlayer(Player player){
```

Comment:
```suggestion
    public void removeOfflinePlayer(@NotNull Player player) {
```
minor nit: style and null checking in signature